### PR TITLE
Preferred Dish UI and other bug fixes

### PR DESCRIPTION
**Pull request by Jen-Lopez**
This PR allows the user to view a description of the dish and be able to reset their "search keyword". Additionally, it filters out the menu item # in the dish name e.g. `01. Fried chicken`

**NOTE**: Should be merged in after #107 

Screen Shot 2021-12-01 at 2 39 23 AM

### CODE REVIEW

**Review by psc358 — DISMISSED** [review]

File: front-end/src/pages/PreferredDish.css [hunk 1]
```diff
@@ -7,17 +7,24 @@
...
+    width: 100%;
     color: #666262;
 }
 
 .pref-dish-row input {
     flex: 15%;
 }
 
+.pref-dish-row:hover, .activate-color {
+    background-color: #fbf1e6!important;
```
Comment (yunko1803) [inline front-end/src/pages/PreferredDish.css:21]: space

File: front-end/src/pages/PreferredDish.css [hunk 2]
```diff
@@ -50,14 +57,37 @@
...
+    transition: 0.5s;
+    overflow-y: auto;
+}
+
+.PreferredDish .content p {
+    padding: 0 5px;
+}
+
+.PreferredDish .active {
+    height: 65px!important;
```
Comment (yunko1803) [inline front-end/src/pages/PreferredDish.css:80]: space

File: front-end/src/pages/PreferredDish.jsx [hunk 3]
```diff
@@ -109,13 +111,35 @@ function PreferredDish() {
...
+                  <div className={isActive && i == activeItem ? "content active" : "content"}>
+                    <p>{dish.description}</p>
+                  </div>
+
+                </div>
             ))}
           </div>
           <Button id = "select-btn" text="Select" width="350px" height="40px" onClick={submitOptions}/>
+          <Link to="/preferences">
+              <Button text = "New Search" width="350px" height="40px" bg="#d1c8c8" onClick={()=> {
```
Comment (yunko1803) [inline front-end/src/pages/PreferredDish.jsx:138]: why not using css?

**Review by yunko1803 — COMMENTED** [review]
slight change!

File: front-end/src/pages/PreferredDish.jsx (hunk 3 above)
Comment (Jen-Lopez) [reply front-end/src/pages/PreferredDish.jsx:138]: The button component has default styling

**Review by thomastai1666 — APPROVED** [review]

**Pull request merged**